pwnlib/asm.py

```diff
@@ -206,6 +206,9 @@ def which_binutils(util, check_version=False):
     if platform.system() == 'Darwin':
         utils = ['g'+util, util]
 
+    if platform.system() == 'Windows':
+        utils = [util + '.exe']
```

Comment:
I remember there was some envvar deifining executable extensions (.bat, .vbs, ...), wasn't there?

Reply:
Yes `PATHEXT`, but I don't think those are relevant here since the tools are always executables. This would require some extensive globbing which I don't think is worth it here.